mlir/include/mlir/Dialect/Func/Transforms/Passes.h

```diff
@@ -29,6 +29,9 @@ namespace func {
 #define GEN_PASS_DECL
 #include "mlir/Dialect/Func/Transforms/Passes.h.inc"
 
+/// Creates type attributes for func parameters, that mirror the actual type
```

Comment:
If you remove `let constructor = "mlir::func::createAnnotateInputTypesPass()";` from the tablegen file, then a constructor will be auto-generated and included by 
```
#define GEN_PASS_DECL
#include "mlir/Dialect/Func/Transforms/Passes.h.inc"
```
above.

Reply:
TIL